com.github.alainm23.planner.json

```diff
@@ -0,0 +1,94 @@
+{
+  "app-id": "com.github.alainm23.planner",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
+  "base": "io.elementary.BaseApp",
+  "base-version": "juno-19.08",
+  "command": "com.github.alainm23.planner",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=xdg-data",
```

Comment:
```suggestion
```
Flatpak gives you access to `~/.var/app/$app-id/{cache,config,data}/`. You don't need to do anything to make it work

Reply:
I tried I but I can't get Planner to access the database file.

Reply:
As I said, replace this https://github.com/alainm23/planner/blob/master/src/Services/Database.vala#L54
with a https://valadoc.org/glib-2.0/GLib.Environment.get_user_data_dir.html
Should be something like 
```
        db_path = Environment.get_user_data_dir () + "/database.db";
```